src/commands/General/guide.ts

```diff
@@ -1,53 +1,71 @@
-import { Emojis } from '#utils/constants';
-import { defaultEmbed } from '#utils/embed';
-import { CLIENT_NAME } from '#utils/environment';
-import { ApplicationCommandRegistry, container } from '@sapphire/framework';
-import { applyLocalizedBuilder, resolveKey, type Target } from '@sapphire/plugin-i18next';
-import { ActionRowBuilder, ButtonBuilder, EmbedBuilder } from 'discord.js';
+import { BrandingColors, Emojis } from '#utils/constants';
+import { ApplicationCommandRegistry } from '@sapphire/framework';
+import { applyLocalizedBuilder, type TFunction } from '@sapphire/plugin-i18next';
+import { ActionRowBuilder, ButtonBuilder, ButtonStyle, EmbedBuilder } from 'discord.js';
 
 export class GuideCommand extends BirthdayyCommand {
 	public override async registerApplicationCommands(registry: ApplicationCommandRegistry) {
 		registry.registerChatInputCommand((builder) =>
-			applyLocalizedBuilder(builder, 'commands/guide:guide').setDMPermission(true)
+			applyLocalizedBuilder(builder, 'commands/guide:name', 'commands/guide:description').setDMPermission(true)
 		);
 	}
 
 	public override async chatInputRun(interaction: BirthdayyCommand.Interaction) {
-		const components = [
+		const t = getSupportedUserLanguageT(interaction);
+
+		const embed = new EmbedBuilder()
+			.setTitle(t('commands/guide:embedTitle'))
+			.setDescription(t('commands/guide:embedDescription'))
+			.setColor(BrandingColors.Primary)
+			.addFields(this.getStartedField(t), this.getConfigField(t), this.getImportantField(t));
+
+		const components = this.createComponents(t);
+
+		return interaction.reply({ embeds: [embed], components, ephemeral: true });
+	}
+
+	private getStartedField(t: TFunction) {
+		const command = BirthdayApplicationCommandMentions.Set;
+		return {
+			name: t('commands/guide:embedFieldsStartedTitle'),
+			value: t('commands/guide:embedFieldsStartedValue', { command })
+		};
+	}
+
+	private getConfigField(t: TFunction) {
+		const commandView = ConfigApplicationCommandMentions.View;
+		const commandEdit = ConfigApplicationCommandMentions.Edit;
+		return {
+			name: t('commands/guide:embedFieldsConfigTitle'),
+			value: t('commands/guide:embedFieldsConfigValue', { commandView, commandEdit })
+		};
+	}
+
+	private getImportantField(t: TFunction) {
+		return {
+			name: t('commands/guide:embedFieldsImportantTitle'),
+			value: t('commands/guide:embedFieldsImportantValue')
+		};
+	}
+
+	private createComponents(t: TFunction) {
+		return [
 			new ActionRowBuilder<ButtonBuilder>().setComponents(
-				await docsButtonBuilder(interaction),
-				await inviteSupportDiscordButton(interaction)
+				new ButtonBuilder()
+					.setLabel(t('commands/guide:buttonDocsLabel'))
+					.setURL('https://birthdayy.xyz/docs')
+					.setStyle(ButtonStyle.Link)
+					.setEmoji(Emojis.Book),
+				new ButtonBuilder()
+					.setURL('https://join.birthdayy.xyz')
+					.setLabel(t('commands/guide:buttonInviteLabel'))
+					.setStyle(ButtonStyle.Link)
```

Comment:
Maybe we should put them in a separate file so that we don't have to repeat our code when reusing this button!

Reply:
It's done here => https://github.com/BirthdayyBot/BirthdayyBot/pull/565/commits/c5a52417d002db768a5551a98c40ebe2900482ef